randovania/games/planets_zebeth/logic_database/Brinstar.txt

```diff
@@ -563,27 +550,20 @@ Extra - map_name: Brinstar/x344_y113
 > Door to Corridor 6; Heals? False
   * Layers: default
   * Normal Door to Corridor 6/Door to Bomb Block shaft (Right)
-  > Bridge
+  > Door to Bomb Block shaft connecting room
       Trivial
 
 > Door to Bomb Block shaft connecting room; Heals? False
   * Layers: default
   * Normal Door to Bomb Block shaft connecting room/Door to Bomb Block shaft (Right)
-  > Bridge
+  > Door to Corridor 6
       Trivial
+  > Door to Cavernous hall
+      Can Use Bombs
 
 > Door to Cavernous hall; Heals? False
   * Layers: default
   * Normal Door to Cavernous hall/Door to Bomb Block shaft (Right)
-  > Bridge
-      Hi-Jump Boots or Ice Beam
-
-> Bridge; Heals? False
-  * Layers: default
-  > Door to Corridor 6
-      Trivial
   > Door to Bomb Block shaft connecting room
-      Trivial
-  > Door to Cavernous hall
-      Can Use Bombs
+      Hi-Jump Boots or Ice Beam
```

Comment:
should ice beam only be locked behind ice beam abuse?

Reply:
Not sure what qualifies as Ice Beam Abuse since the game expects you to use Ice Beam to stand on enemies.